src/Miunie.Core/Services/ProfileService.cs

```diff
@@ -14,9 +14,14 @@ public ProfileService(IDiscordMessages discordMessages)
 
         public async Task ShowProfile(MiunieUser u, MiunieChannel c)
         {
-            // TODO(Peter): This should probably come from some kind of a
-            // Language Service to enable things like translation and easy
-            // edits in case of a misspell.
+            // TODO(Charly): Replace the following code with the commented code,
+            // once the new service collection is implemented and the discord
+            // wrapper updated to v4:
+
+            // var rep = u.Reputation;
```

Comment:
This explaining variable doesn't make it any clearer since it's just used on the next line.
If 80 characters was the issue, just collapse the `GetFormatted` method call.